packages/allure-jest/src/AllureJestApi.ts

```diff
@@ -59,13 +59,13 @@ export class AllureJestApi implements AllureRuntimeApiInterface {
     });
   }
 
-  attachment(content: string | Buffer, type: string): void {
+  attachment(content: string | Buffer, type: string, name?: string): void {
     const isBuffer = Buffer.isBuffer(content);
 
     this.sendMetadata({
       attachments: [
         {
-          name: "Attachment",
+          name: name ? name : "Attachment",
```

Comment:
Let's make it shorter `name || "Attachment"`

Reply:
done